backend/app/views/spree/admin/orders/_risk_analysis.html.erb

```diff
@@ -1,50 +1,26 @@
-<fieldset class="no-border-bottom" id="risk_analysis">
-  <legend><%= "#{t('spree.risk_analysis')}: #{t('spree.not') unless @order.is_risky?} #{t('spree.risky')}" %></legend>
+<details id="risk_analysis">
+  <summary><%= t('spree.risk_analysis') %>: <%= t('spree.risky') %></summary>
```

Comment:
As a side note, if you say the translation very fast, it sounds like "spreeesky" :laughing: